locations/dict_parser.py

```diff
@@ -291,13 +291,29 @@ def parse(obj: dict) -> Feature:
         item["ref"] = DictParser.get_first_key(obj, DictParser.ref_keys)
         item["name"] = DictParser.get_first_key(obj, DictParser.name_keys)
 
+        geojson_geometry_exists = obj.get("geometry") and obj["geometry"].get("type") in [
+            "Point",
+            "MultiPoint",
+            "LineString",
+            "MultiLineString",
+            "Polygon",
+            "MultiPolygon",
+        ]
         if (
-            obj.get("geometry")
-            and obj["geometry"].get("type")
-            in ["Point", "MultiPoint", "LineString", "MultiLineString", "Polygon", "MultiPolygon"]
+            geojson_geometry_exists
             and isinstance(obj["geometry"].get("coordinates"), list)
+            and len(obj["geometry"]) == 2
         ):
             item["geometry"] = obj["geometry"]
+        elif (
+            geojson_geometry_exists
+            and isinstance(obj["geometry"].get("coordinates"), tuple)
+            and len(obj["geometry"]) == 2
+        ):
+            # Necessary for use with geopands.read_file which returns
+            # positions as tuples not lists. Convert tuples to lists.
+            obj["geometry"]["coordinates"] = list(obj["geometry"]["coordinates"])
+            item["geometry"] = obj["geometry"]
```

Comment:
What part of this was failing with tuples?

Reply:
Pipelines such as `locations/pipelines/geojson_multipoint_simplification.py` were doing type checks against `Feature["geometry"]["coordinates"][0]` assuming it was always a list (as usually written in GeoJSON files) and would not detect the Python alternative of storing GeoJSON positions as tuples of two coordinates not lists of two coordinates.

I figure it'd be easier and less error prone for ATP pipelines, exporters, etc to only have to deal with GeoJSON positions (per `Feature["geometry"]["coordinates"][0]`) as a single Python type, rather than two or more types.